scripts/fetchReferrals.ts

```diff
@@ -63,11 +62,10 @@ async function main() {
   )
   const uniqueEvents = removeDuplicates(referralEvents)
   const builderAllowList = args.builderAllowList
-    ? (parse(readFileSync(args.builderAllowList, 'utf-8').toString(), {
-        skip_empty_lines: true,
-        delimiter: ',',
-        columns: true,
-      }) as Address[])
+    ? readFileSync(args.builderAllowList, 'utf-8')
```

Comment:
Can we add a single column header instead to match the format of the other CSV?  I think it would be `referrerId` (not the best name at this point, but we can rename everything in one pass)